### ISSUE TITLE
Menu Changes when Unstuck and when Stuck

### ISSUE THREAD

**Issue by jotcreative** [pojome/elementor#5813]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
Sometimes a menu looks great with tall margins and a transparent background over a hero image. But if you use it as a sticky menu, tall and transparent don't work. When you scroll down and it sticks, it should be shorter and with a solid color behind it.

**Describe the solution you'd like**
I think one set of options for the plain menu, and then one set of options for the sticky menu, and then a smooth transition as you scroll down would be simple and effective!

**Describe alternatives you've considered**
I'd hate to have to get a third party plugin, but I guess if I had to... :(

**Additional context**

**Comment by silvioghedin** [comment]
+1

**Comment by silvioghedin** [comment]
+1

**Comment by enviedbymost** [comment]
> @jotcreative
> 
> Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.
> 
> For now, this is possible only via Custom CSS.
> 
> Thanks for the feedback.

Well the 'large demand' need somewhere to post their desire for it. 

Why's the thread being closed? lol

**Comment by jotcreative** [comment]
Ironically, I just realized that the Astra theme already had this capability - I lost it by switching to a header template - Elementor's "premium" feature!

**Comment by qwik3r** [comment]
Agreed. I am using Astra as well and it's transparent header element does not work with Elementor. Having this built in would be great.

**Comment by jotcreative** [comment]
Where can we post this to demonstrate demand? I really want to use Elementor for my header, but as long as I can't have the logo/menu hover over my hero image and then turn into a solid bar after the user scrolls down, I have to use the much less powerful Astra sticky menu (which is cosmetically perfect).

Here is your demand: someone is already coding a 3rd party solution, which I don't want to have to install: https://wordpress.org/plugins/sticky-header-effects-for-elementor/

**Comment by elasticeye** [comment]
+1 for this. 
A different menu when scrolling is a must.
